app-backend/batch/src/main/scala/com/azavea/rf/batch/export/spark/Export.scala

```diff
@@ -208,7 +208,7 @@ object Export extends SparkJob with Config with LazyLogging {
     conf: HadoopConfiguration
   ): Unit = {
     def path(key: SpatialKey): ExportDefinition => String = { ed =>
-      s"${ed.output.getURLDecodedSource}/${ed.input.resolution}-${key.col}-${key.row}-${ed.id}.tiff"
+      s"/${ed.output.getURLDecodedSource}/${ed.input.resolution}-${key.col}-${key.row}-${ed.id}.tiff"
```

Comment:
This line is where I nearly went insane, since the dropbox SDK kept throwing an error that just said `String 'path' does not match the pattern`

WHAT PATTERN DROPBOX WHY DID YOU WRITE THAT :cry: I wasn't even sending `'path'` anywhere